tools/build.py

```diff
@@ -52,7 +52,7 @@ def legacy_kern_table(ufo):
 
 def flatten_kerning(ufo, key_glyphs_only=False):
     kerning = {}
-    for key, value in ufo.kerning.items():
+    for key, value in ufo.kerning.items(): # [(l-hand, r-hand): value, ...]
```

Comment:
This can probably be rewritten as:
```python
for (left, right), offset in ufo.kerning.items():
```
Or `(first, second)` etc to avoid terms that might be confusing for left-to-right support.